app/pb_kits/playbook/pb_textarea/_textarea.scss

```diff
@@ -2,9 +2,9 @@
 @import "./textarea_mixin";
 
 [class^=pb_textarea_kit] {
+  margin: 8px;
   margin-bottom: $space_sm;
   .pb_textarea_kit_label {
-    margin-bottom: $space_xs;
```

Comment:
@christinaatai We noticed this had different values when comparing it to the text input kit. So we changed this to match. We also noticed that the font does not match for these two kits.
<img width="1297" alt="Screen Shot 2019-12-11 at 1 27 55 PM" src="https://user-images.githubusercontent.com/35009869/70649995-3b6eca80-1c1c-11ea-8260-5d51909ffa7b.png">

Thoughts on adjusting the margin and removing the margin bottom?

Should we make the font for the text area match the font used for the text inputs?

Reply:
@roxannecojocariu Looking into this deeper the text input kit is actually incorrect. We need to remove this line on the text input kit: https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/pb_text_input/_text_input.scss#L5

& this file should remain the same

Reply:
@jasoncypret Thoughts on the text area and text input font differences?

Reply:
@terryfinn Working with Katie, we noticed a change was merged to master to fix the difference in fonts.